### DISCUSSION TITLE
next-i18next not working

### DISCUSSION THREAD

**Discussion by Amin-bash** [vercel/next.js#17414] [Help] [answered]
every time it shows a different error for example it keeps redirected me to localhost:3000/en , namespacesRequired can not be defined and so on this is the link to the repo anyone can help??
https://github.com/Amin-bash/taiba-project

**Comment by lfades** [comment, marked as answer]
Consider reporting issues with `next-i18next` in their repo: https://github.com/isaachinman/next-i18next.
